framework/test/unit/modules/token/method.spec.ts

```diff
@@ -1223,4 +1223,22 @@ describe('token module', () => {
 			});
 		});
 	});
+
+	describe('escrowSubstoreExists', () => {
+		const escrowChainID = defaultForeignTokenID.slice(0, CHAIN_ID_LENGTH);
+
+		it('should return false if escrow subStore does not exist', async () => {
```

Comment:
```suggestion
		it('should return false if escrow subStore does not exist for the given chain id and token id', async () => {
```
minor optional suggestion